socorro/unittest/external/es/test_crashstorage.py

```diff
@@ -179,65 +177,27 @@ def test_index_crash_indexable_keys(self):
         )
 
         # Ensure that the document was indexed by attempting to retreive it.
-        doc = self.es_client.get(
-            index=self.config.elasticsearch.elasticsearch_index,
+        doc = self.conn.get(
+            index=self.es_context.get_index_for_date(utc_now()),
             id=processed_crash["uuid"],
         )
 
         # Verify keys that aren't in super_search_fields aren't in the raw or processed
         # crash parts
         raw_crash = doc["_source"]["raw_crash"]
-        assert raw_crash == {"BuildID": "20200506000000"}
```

Comment:
These assertions are testing keys and whether keys that should be removed were removed. I changed the code to ignore the values.